src/xunit.performance.analysis/Program.cs

```diff
@@ -192,10 +210,32 @@ private static List<TestResultComparison> DoComparisons(List<Tuple<string, strin
 
                 foreach (var metric in iteration.MetricValues)
                 {
+                #if !LINUX_BUILD
```

Comment:
maybe it would be better to just have a different SummarizeTestResults method for Linux builds, rather than the same method with many #if inside